api/src/lib/aws.ts

```diff
@@ -154,7 +153,6 @@ async function sendSqsMessage(queueUrl: string, messageBody: any) {
   )
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This function's signature doesn't appear to have any unused vars, so removed this pragma.